src/components/ClaimReview/form/assignedForm.ts

```diff
@@ -0,0 +1,49 @@
+const assignedForm = {
```

Comment:
This is going to be a function that receives the translation object in order to apply i18n.

Reply:
When we turn into a function, it's returning a function, not a object to param formUi